LeadKit/LeadKit/Extensions/UIImage/UIImage+Cropping.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UIImage+Cropping.swift
+//  LeadKit
+//
+//  Created by Николай Ашанин on 28.09.16.
+//  Copyright © 2016 Touch Instinct. All rights reserved.
+//
+
+import UIKit
+
+public extension UIImage {
+
+    /**
+     creates a cropped copy of an image.
+
+     - parameter bounds: The bounds of the rectangle inside the image.
+
+     - returns: A new image
+     */
+    public func crop(bounds: CGRect) -> UIImage? {
+        guard let cgImage = CGImageCreateWithImageInRect(CGImage, bounds) else {
```

Comment:
why not https://developer.apple.com/reference/coregraphics/cgimage/1454683-cropping ?